<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>658</width>
                <height>597</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>提取特征词</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
                <widget class="QFrame" name="frame">
                    <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_2">
                        <item>
                            <widget class="QLabel" name="label">
                                <property name="text">
                                    <string>可以多选</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QListWidget" name="listWidget_colnames">
                                <property name="selectionMode">
                                    <enum>QAbstractItemView::ExtendedSelection</enum>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QFrame" name="frame_4">
                    <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_3">
                        <property name="spacing">
                            <number>7</number>
                        </property>
                        <property name="leftMargin">
                            <number>11</number>
                        </property>
                        <property name="topMargin">
                            <number>11</number>
                        </property>
                        <property name="rightMargin">
                            <number>11</number>
                        </property>
                        <property name="bottomMargin">
                            <number>11</number>
                        </property>
                        <item>
                            <widget class="QLabel" name="label_4">
                                <property name="text">
                                    <string>词性选择【可以多选】</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QListWidget" name="listWidget_speech">
                                <property name="selectionMode">
                                    <enum>QAbstractItemView::ExtendedSelection</enum>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QFrame" name="frame_2">
                    <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout">
                        <item>
                            <widget class="QFrame" name="frame_3">
                                <property name="frameShape">
                                    <enum>QFrame::StyledPanel</enum>
                                </property>
                                <property name="frameShadow">
                                    <enum>QFrame::Raised</enum>
                                </property>
                                <layout class="QHBoxLayout" name="horizontalLayout_2">
                                    <item>
                                        <widget class="QLabel" name="label_2">
                                            <property name="text">
                                                <string>特征词数量:</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QSpinBox" name="spinBox_threshold">
                                            <property name="value">
                                                <number>5</number>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="label_3">
                                <property name="text">
                                    <string>会使用到停用词表和受控词表</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="pushButton">
                                <property name="styleSheet">
                                    <string notr="true">color: rgb(255, 255, 255); background-color: rgb(28, 177,
                                        245);
                                    </string>
                                </property>
                                <property name="text">
                                    <string>OK</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
